README.md

```diff
@@ -10,44 +10,228 @@ IO operations tool
+
+  --service_metadata.description=service_metadata.description  Description of the sevice
+
+  --service_metadata.email=service_metadata.email              Email of the institution
+
+  --service_metadata.pec=service_metadata.pec                  Pec of the institution
+
+  --service_metadata.phone=service_metadata.phone              Phone number
+
+  --service_metadata.privacy_url=service_metadata.privacy_url  Privacy url
+
+  --service_metadata.scope=NATIONAL|LOCAL                      Scope of the service can be NATIONAL or LOCAL
+
+  --service_metadata.tos_url=service_metadata.tos_url          Term of Service url
+
+  --service_metadata.web_url=service_metadata.web_url          Url of the service
+
+  --service_name=service_name                                  (required) Name of the service
 
-- [`io-ops hello`](#io-ops-hello)
-- [`io-ops help [COMMAND]`](#io-ops-help-command)
-- [`io-ops messages:attributes`](#io-ops-messagesattributes)
-- [`io-ops messages:check-content`](#io-ops-messagescheck-content)
-- [`io-ops messages:list FISCALCODE`](#io-ops-messageslist-fiscalcode)
-- [`io-ops profiles:delete FISCALCODE`](#io-ops-profilesdelete-fiscalcode)
-- [`io-ops profiles:exist`](#io-ops-profilesexist)
-- [`io-ops profiles:list`](#io-ops-profileslist)
-- [`io-ops services:details`](#io-ops-servicesdetails)
-- [`io-ops services:list`](#io-ops-serviceslist)
+  --version=version                                            Version of the service
+
+EXAMPLE
+  $ io-ops api-service:create  --department_name=department_test --organization_fiscal_code=12345670000  
+  --organization_name=organization_name --service_id=test-api-service --service_name=test_api-service
+```
+
+_See code: [src/commands/api-service/create.ts](https://github.com/teamdigitale/io-ops/blob/v0.2.0/src/commands/api-service/create.ts)_
+
+## `io-ops api-service:get SERVICEID`
+
+Get the service by serviceId
+
+```
+USAGE
+  $ io-ops api-service:get SERVICEID
+
+ARGUMENTS
+  SERVICEID  id of the service
+```
+
+_See code: [src/commands/api-service/get.ts](https://github.com/teamdigitale/io-ops/blob/v0.2.0/src/commands/api-service/get.ts)_
+
+## `io-ops api-service:get-all`
+
+Get all services
+
+```
+USAGE
+  $ io-ops api-service:get-all
+```
+
+_See code: [src/commands/api-service/get-all.ts](https://github.com/teamdigitale/io-ops/blob/v0.2.0/src/commands/api-service/get-all.ts)_
+
+## `io-ops api-service:keys SERVICEID`
+
+Get subscription keys associated to service
+
+```
+USAGE
+  $ io-ops api-service:keys SERVICEID
+
+ARGUMENTS
+  SERVICEID  id of the service
+```
+
+_See code: [src/commands/api-service/keys.ts](https://github.com/teamdigitale/io-ops/blob/v0.2.0/src/commands/api-service/keys.ts)_
+
+## `io-ops api-service:logo SERVICEID`
+
+Update service data with base64 of the logo
+
+```
+USAGE
+  $ io-ops api-service:logo SERVICEID
+
+ARGUMENTS
+  SERVICEID  id of the service
+
+OPTIONS
+  --logo=logo  (required) Path of logo image to be converter into base64
```

Comment:
``
```suggestion
  --logo=logo  (required) Path of logo image to be converted into base64
```

it is not clear on why the fact that the logo gets converted into base64 is an useful info for the cli user.
I'd rather state that the logo gets uploaded.

Reply:
sure!